ififuncs.py

```diff
@@ -730,11 +730,11 @@ def get_user():
     user = ''
     if user not in ('1', '2', '3', '4', '5', '6', '7', '8', '9'):
         user = input(
-            '\n\n**** Who are you?\nPress 1,2,3,4,5,6,7,8,9\n\n1. Aoife Fitzmaurice\n2. Gavin Martin\n3. Kieran O\'Leary\n4. Raelene Casey\n5. Noelia Romero\n6. Ana Truchanova\n7. Eoin O\'Donohoe\n8. Yujing Huang\n9. Colm Connolly\n'
+            '\n\n**** Who are you?\nPress 1,2,3,4,5,6,7,8,9\n\n1. Aoife Fitzmaurice\n2. Gavin Martin\n3. Kieran O\'Leary\n4. Raelene Casey\n5. Noelia Romero\n6. Ana Truchanova\n7. Eoin O\'Donohoe\n8. Yujing Huang\n9. Colm Connolly\n10. Yazhou He\n11. Aoife McGrath\n'
```

Comment:
Also just up to 9

Reply:
From a display perspective, having the numbers at the start going up to 11 should help '\n\n**** Who are you?\nPress 1,2,3,4,5,6,7,8,9\n\n1.